2.cpp

```diff
@@ -0,0 +1,50 @@
+#include <fstream>
+#include <string>
+#include <iostream>
+#include <cstdio>
+#include <vector>
+
+using namespace std;
+ 
+int main()
+{
+	string inputarr[3];
+	ifstream inp("doc2.txt");
+	
+	/*if (in == NULL)
+	{
+		perror("Ошибка открытия файла");
+	}
+	else
+	{*/
+	char str[17];
+	/*for(int i=0;i<sizeof(inputarr)/sizeof(inputarr[0]);i++)
+		{
+			
+			do
+			{
+
+				temp=puts(str);
+				temp=temp;
+			}
+			while(" " not in temp);
+			inputarr[i]=str;
+		}
+		*/
+	for(int j=0;j<sizeof(inputarr)/sizeof(inputarr[0]);j++)
```

Comment:
What if the size of the file is bigger than 3 words? Or, what if a size of a single word is 3GB?